btree/btree.c

```diff
@@ -3756,10 +3778,6 @@ int64_t btree_destroy_tick(struct m0_sm_op *smop)
 		 */
```

Comment:
It seems you do not need any switch-case statement. You can instead just have an assert to make sure that the state when you enter is P_INIT and then the rest of the code can be without the existing switch-case.

Reply:
Ok.